domain-server/resources/web/wizard/js/wizard.js

```diff
@@ -5,7 +5,7 @@ var Metaverse = {
 var currentStepNumber;
 
 $(document).ready(function(){
-  Strings.ADD_PLACE_NOT_CONNECTED_MESSAGE = "You must have an access token to query your High Fidelity places.<br><br>" +
+  Strings.ADD_PLACE_NOT_CONNECTED_MESSAGE = "You must have an access token to query your Metaverse places.<br><br>" +
     "Please go back and connect your account.";
 
   $('#connect-account-btn').attr('href', URLs.METAVERSE_URL + "/user/tokens/new?for_domain_server=true");
```

Comment:
this needs to be changed to the page in static that does this functionality or rename that page to this address

Reply:
Are you talking about line 11? I'm fixing that now.